### DISCUSSION TITLE
Udemy Course

### DISCUSSION THREAD

**Discussion by buildwithedward** [mrdbourke/pytorch-deep-learning#77] [Q&A] [answered]
Hi Daniel, When can we expect this course on Udemy? Is there a possibility in the future?

**Comment by mrdbourke** [comment, marked as answer]
Hey @edwardcodes, 

No timeline for Udemy yet but will likely be when all the materials are finished (as of this comment I've got section 08 and 09 to go).

**Comment by mrdbourke** [comment]
Hi @edwardcodes,

Good news, the course is available in full on Udemy!

You can see it here: https://www.udemy.com/course/pytorch-for-deep-learning/?couponCode=ZTMGOODIES7 (discount code "ZTMGOODIES7" applied for a big discount over the next few days!)

**Reply by buildwithedward** [reply]
Hi Daniel,

Thanks for the update. I bought the course as soon as I got your mail. Keep doing the good work. Expecting more valuable content from you.

**Reply by mrdbourke** [reply]
Thank you Edward! 

Enjoy my friend!

**Comment by nejadmanesh** [comment]
thank you sir . your coursews very use full for  me!

**Comment by ytanikin** [comment]
Is the course up to date to pytorch 2?
